.gitignore

```diff
@@ -1,3 +1,5 @@
+inst/
+*~
```

Comment:
What is intent here? I have never seen this type if entry/files. OS specific?

Reply:
The first line prevents adding inst/ folder which is often used only for local purposes. But not always. We can decide to share this, I can remove it from .gitignore or just keep it locally with no reference in .gitignore

I realize that the second line is OS specific. Can be removed.

Reply:
`inst` is valid R package folder, so it can end up in the repo.
I was wondering just about the second line

Reply:
Ok. The second line should not cause harm so I propose we keep it. And will remove inst/